### PR TITLE
#257-desc-03-view-recent-transactions-on-home-page

### PR DESCRIPTION
**Pull request by SokhibJones**

### CODE REVIEW

File: src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs [hunk 1]
```diff
@@ -16,6 +16,15 @@ public class TransactionController : ControllerBase
         private readonly IMediator _mediator;
         public TransactionController(IMediator mediator) => _mediator = mediator;
 
+        [HttpGet("Recents")]
+        public async Task<IActionResult> GetTopRecents(CancellationToken cancellationToken)
+        {
+            var userId = Guid.Parse(User.FindFirst("UserId").Value);
+            var result = await _mediator.Send(new GetTop10RecentTransactionsQuery(userId), cancellationToken);
+
```
Comment (Saigunas) [inline src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs:24]: I don't think the amount of transactions should be hard-coded, better pass a value.

File: src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs [hunk 2]
```diff
@@ -1,27 +1,15 @@
-﻿using BudgetManager.Model.Enums;
-
-namespace BudgetManager.CQRS.Responses.TransactionResponses
+﻿namespace BudgetManager.CQRS.Responses.TransactionResponses
 {
     public class TransactionResponse
     {
```
Comment (Saigunas) [inline src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs:4]: Category type should also be included.

Comment (SokhibJones) [reply src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs:4]: There's actually a property called CategoryName, which is the string representation of OperationType enum.

File: src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs (hunk 1 above)
Comment (SokhibJones) [reply src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs:24]: I don't know but in user stories, it asks to display ten recent transaction records.

Comment (Saigunas) [reply src/BudgetManager/BudgetManager.API/Controllers/V1/TransactionController.cs:24]: It is just a better practice to not hard-code it in the name, at least pass 10 by default in that case

File: src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs (hunk 2 above)
Comment (Saigunas) [reply src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs:4]: CategoryType is responsible for enum, CategoryName is for example "Shopping".

**Pull request closed without merging**